meta-dts-distro/recipes-gnome/gcab/gcab_%.bbappend

```diff
@@ -0,0 +1 @@
+do_package_qa[noexec] = "1"
```

Comment:
Can you use `insane_skip+=buildpaths` as in wolfssl so we don't need to disable whole qa check?

Reply:
647703793a3ae0b2f7bbc95a0f15d6407b6cff04